src/components/Chips/Chips.tsx

```diff
@@ -66,13 +65,9 @@ interface ChipsProps extends VibeComponentProps {
    */
   ariaLabel?: string;
   /**
-   * Should element be focusable & clickable - for backward compatability
+   * If you are using onMouseDown or onClick but do not want your chip to be used as a button (in terms of appearance or aria attributes)
```

Comment:
what do you mean by "appearance"

Reply:
chips color is change while hovering for now